PythonAPI/pycocotools/cocoeval.py

```diff
@@ -432,8 +432,8 @@ def _summarize( ap=1, iouThr=None, areaRng='all', maxDets=100 ):
             iouStr = '{:0.2f}:{:0.2f}'.format(p.iouThrs[0], p.iouThrs[-1]) \
                 if iouThr is None else '{:0.2f}'.format(iouThr)
 
-            aind = [i for i, aRng in enumerate(p.areaRngLbl) if aRng == areaRng]
-            mind = [i for i, mDet in enumerate(p.maxDets) if mDet == maxDets]
+            aind = p.areaRngLbl.index(areaRng)
+            mind = p.maxDets.index(maxDets)
```

Comment:
That has different semantics though: your code only gives the first match and throws ValueError when nothing matches, whereas the original list allowed for multiple matches and may return an empty list.